From f3664206751341294df0d88e447629eef2bd3c50 Mon Sep 17 00:00:00 2001
From: Dominique Martinet <dominique.martinet@atmark-techno.com>
Date: Wed, 31 Mar 2021 12:22:15 +0900
Subject: [PATCH] introspection/bps.c: remove BPF_MAP_TYPE_TASK_STORAGE

BPF_MAP_TYPE_TASK_STORAGE got added in 5.11 but we are on 5.10 kernel

diff --git a/introspection/bps.c b/introspection/bps.c
index 0eae675e3817..b5595442faee 100644
--- a/introspection/bps.c
+++ b/introspection/bps.c
@@ -78,7 +78,6 @@ static const char * const map_type_strings[] = {
   [BPF_MAP_TYPE_STRUCT_OPS] = "struct_ops",
   [BPF_MAP_TYPE_RINGBUF] = "ringbuf",
   [BPF_MAP_TYPE_INODE_STORAGE] = "inode_storage",
-  [BPF_MAP_TYPE_TASK_STORAGE] = "task_storage",
 };
 
 #define ARRAY_SIZE(x) (sizeof(x) / sizeof(*(x)))
